lib/tracker_api/client.rb

```diff
@@ -285,4 +285,15 @@ def next_page_params
       end
     end
   end
+
+  require 'faraday_middleware/response_middleware'
+  class ParseJsonWithSymbols < FaradayMiddleware::ResponseMiddleware
```

Comment:
`Dry::Struct` expects keys to be symbols when creating the `Dry::Struct` based class from a hash (e.g. parsed JSON).